.changeset/quiet-stingrays-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-azure-devops': minor
+---
+
+The getBuildRuns function now checks contains multiple comma-separated builds and splits them to send multiple requests for each and concatenates the results.
```

Comment:
It would have been great to see an example here, would really help those doing the upgrade